src/common/processing_linux.c

```diff
@@ -20,40 +20,52 @@ static inline void waitpid_wrapper(const pid_t* pid)
         waitpid(*pid, NULL, 0);
 }
 
+static inline int ffPipe2(int *fds, int flags) {
```

Comment:
Please follow the coding style of fastfetch. Move `{` to a new line

Reply:
Fixed.